Core/include/Acts/Propagator/Navigator.hpp

```diff
@@ -35,7 +35,7 @@ namespace Acts {
 template <typename object_t>
 struct NavigationOptions {
   /// The boundary check directive
-  BoundaryTolerance boundaryTolerance = BoundaryTolerance::None();
+   BoundaryTolerance boundaryTolerance = BoundaryTolerance::AbsoluteCartesian{2*UnitConstants::mm,2*UnitConstants::mm};
```

Comment:
This is why the modules get bigger.  However, without handling of multiple modules passing the intersection, this is I suppose counterproductive.

Reply:
Just to clarify and resolve, this is not in this current PR. I've removed this from the navigator